### DISCUSSION TITLE
A real, modern, and responsive credit card management application.

### DISCUSSION THREAD

**Discussion by Macktireh** [TomSchimansky/CustomTkinter#2511] [Showcase]
<p align="center">
  <img src="https://github.com/Macktireh/EasyCardGUIDesktop/raw/main/assets/demo/Screenshot1.png" alt="Logo" height=300>
</p>

<p align="center">
    <strong align="center">EasyCardGUIDesktop</strong>
</p>

Hi,

I am developing a real, modern, and responsive credit card management application for my brother using CustomTkinter.

Here are the various features of the app:

- **Auth**: Offers a secure login mechanism to ensure that only authorized users can access the application.
- **Credit Card Management**: Enables users to manage their credit cards efficiently.
- **Number Extraction**: Uses AI to extract credit card numbers from images.
- **Dashboard**: Displays statistics on the total number of cards, valid and invalid cards.
- **Data Table**: Presents a detailed view of credit card data in a tabular format.
- **Appearance Theme**: Allows users to customize the visual theme of the application, providing options such as a light or dark mode to suit user preferences.

## Demo

To see EasyCardGUIDesktop in action, watch the demonstration video below:

![EasyCardGUIDesktop Demo](https://github.com/Macktireh/EasyCardGUIDesktop/raw/main/assets/demo/Demo-EasyCardGUIDesktop.gif)

Source code link on GitHub: [https://github.com/Macktireh/EasyCardGUIDesktop](https://github.com/Macktireh/EasyCardGUIDesktop) (Feel free to give it a star ⭐)

Thanks

**Comment by BhagyaJyoti22006** [comment]
This is amazing! Hats off man

**Reply by Anonymous6598** [reply]
Hi @BhagyaJyoti22006! Long time no see. How are you?

**Reply by BhagyaJyoti22006** [reply]
> Hi @BhagyaJyoti22006! Long time no see. How are you?

ahoy bro! doing great. hehe

**Reply by Anonymous6598** [reply]
Great to hear!
